src/lib/Components/Inbox/Conversations/Messages.tsx

```diff
@@ -120,11 +123,21 @@ export class Messages extends React.Component<Props, State> {
 
     return (
       <FlatList
-        inverted
-        data={messages}
+        inverted={!this.state.shouldStickFirstMessageToTop}
```

Comment:
Hah, smart 👌 

I do wonder if there isn’t a way to do this without having to measure the height, though. Do you think that there’s a way to use flexbox in such a way that it does _not_ make the `FlatList` take 100% of the parent’s size, but instead (assuming it has its origin in the top-left corner) make it grow from the top down _until_ it reaches the bottom of the parent’s bounds and then do the equivalent of `overflow: scroll`?